kernel-module/build-for-jetson-drive-igpu-native.sh

```diff
@@ -20,4 +20,22 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+KERNEL_SRC_DIR="/usr/src/nvidia/nvidia-oot/"
+if [ ! -d ${KERNEL_SRC_DIR} ]; then
+	echo "Kernel source not found in $KERNEL_SRC_DIR"
+	KERNEL_SRC_DIR="/usr/src/./linux-headers-tegra-oot*"
```

Comment:
I wonder if p2p.h or module symver be ever in this path?

Reply:
Yes both are at same directory for Nvidia OOT.
/usr/src/nvidia/nvidia-oot/include/linux/nv-p2p.h
/usr/src/nvidia/nvidia-oot/Module.symvers

Reply:
I meant is why do you need to search this /usr/src/./linux-headers-tegra-oot* path at all?

Reply:
Ohh, We have differ path for Nvidia OOT on AGX and IGX. So we need to check for both path.
AGX: /usr/src/nvidia/nvidia-oot/
IGX: /usr/src/./linux-headers-tegra-oot